docs/getting-started/installing-geth.md

```diff
@@ -143,6 +143,22 @@ sudo pacman -Sy
 
 When the node is started again, Geth will automatically use all the data from the previous version and sync the blocks that were missed while the node was offline.
 
+### Gentoo via Portage {#gentoo-via-portage}
+
+Geth is included in the Gentoo repository as [`net-p2p/go-ethereum`](https://packages.gentoo.org/packages/net-p2p/go-ethereum). It can be installed by running:
```

Comment:
Is this maintained by you personally, @SamWilsn ? 

IMO we should integrate it into our CI pipeline if we have it here -- no offense meant to you, but we should not rely on specific individuals pushing packages years into the future.

Reply:
> Is this maintained by you personally, @SamWilsn ?

I am the current maintainer, yes. It has been in the Gentoo repository for much longer than that, however. Note that this is the official Gentoo repository, and is not something like a PPA or the AUR.

> IMO we should integrate it into our CI pipeline if we have it here

I'd be happy to investigate automation if there's a chance of merging it. Essentially this would entail copying a file, updating the URL, running a build, and opening a pull request.

How is this implemented for Arch Linux?

Reply:
We don't maintain the ArchLinux package, it's done by a community member.

Reply:
Ah.

Well as far as automation goes, I believe it would go in [`ci.go`](https://github.com/ethereum/go-ethereum/blob/master/build/ci.go)? The process for 1.14.9 would look something like:

1. Clone [`gentoo/gentoo`](https://github.com/gentoo/gentoo).
2. Copy [`net-p2p/go-ethereum/go-ethereum-1.14.8.ebuild`](https://github.com/gentoo/gentoo/blob/master/net-p2p/go-ethereum/go-ethereum-1.14.9.ebuild) to `net-p2p/go-ethereum/go-ethereum-1.14.9.ebuild`.
3. Update the source tarball filename (`LONG_VERSION`).
4. Run a test build with and without `USE=devtools`. I imagine this would use docker.
5. Commit the new ebuild file.
6. Push it and create a pull request.

Should I go ahead and build all of that, or can I continue to do it by hand (with the assumption it'll get removed from the documentation if the package gets out of date)?

Reply:
I don't think we want to automate this in our pipeline. `ci.go` is the build script that creates binaries. We do have some automation in there for Debian/Ubuntu as well. But it's just about generating packaging files.

Reply:
I see three options for moving this PR forward:

- Close this pull request without merging, even though neither Arch nor FreeBSD have automation (I'm fine with this, but I'd like geth to be explicit about it.)
- Merge the pull request as is (suffers from single point of failure: me.)
- I build automation outside the go-ethereum repository (same single point of failure concern.)

How would you like to proceed?